x-pack/plugins/case/server/routes/api/cases/find_cases.ts

```diff
@@ -37,17 +38,14 @@ export function initFindCasesApi({ caseService, caseConfigureService, router }:
           CasesFindRequestRt.decode(request.query),
           fold(throwErrors(Boom.badRequest), identity)
         );
-
         const queryArgs = {
           tags: queryParams.tags,
           reporters: queryParams.reporters,
           sortByField: queryParams.sortField,
-          status: queryParams.status,
+          status: queryParams.status === AllCaseType ? undefined : queryParams.status,
```

Comment:
What happens if the status is `undefined`. Does the backend strips out the variable automatically?

Reply:
we use status to build query statement, if it is undefined, it won't appear in the statement.

Reply:
Hmm could we instead not pass `status` to the backend at all? If we don't pass `status` it will be undefined and the backend will return everything anyway.